packages/sd-jwt-vc/src/index.ts

```diff
@@ -34,4 +46,79 @@ export class SDJwtVcInstance extends SDJwtInstance<SdJwtVcPayload> {
       }
     }
   }
+
+  /**
+   * Fetches the status list from the uri.
+   * @param uri
+   * @returns compact JWT
+   */
+  private async statusListFetcher(uri: string): Promise<string> {
+    // according to the spec we assume according to the spec that the response is a compact JWT
+    return fetch(uri).then((res) => res.text());
+  }
```

Comment:
I think it would be good to add a timeout to this. It would also be good to e.g. add handling for large response size (so if the body exceeds 1MB we will cancel the fetch to prevent a very large payload from crashing the system). But that can be added later / is maybe better suited for a custom implementation (but it does affect security so that's something that should be taken into account.

Also the method didn't take into account non 200 responses in which case response.text is probably not a JWT

```suggestion
/**
 * Fetches the status list from the uri with a timeout of 10 seconds.
 * @param uri The URI to fetch from.
 * @returns A promise that resolves to a compact JWT.
 */
private async statusListFetcher(uri: string): Promise<string> {
  const controller = new AbortController();
  const timeoutId = setTimeout(() => controller.abort(), 10000);

  try {
    const response = await fetch(uri, { signal: controller.signal });
    clearTimeout(timeoutId);
    if (!response.ok) {
      throw new Error(`Error fetching status list: ${response.status} ${await response.text()}`);
    }
    
    return await response.text();
  } catch (error) {
    clearTimeout(timeoutId);
    throw error;
  }
}
```

Reply:
Yeah agreed! LGTM